src/ViewRendererBundle/Bridge/Twig/Loader/TwigStringLoader.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace ChameleonSystem\ViewRendererBundle\Bridge\Twig\Loader;
+
+/**
+ * Loads a template from a string.
+ *
+ * This loader should NEVER be used. It only exists for Twig internal purposes.
+ *
+ * When using this loader with a cache mechanism, you should know that a new cache
+ * key is generated each time a template content "changes" (the cache key being the
+ * source code of the template). If you don't want to see your cache grows out of
+ * control, you need to take care of clearing the old cache file by yourself.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ *
+ * In the Chameleon System this class is only used in case a template could not be found. We can then display the
```

Comment:
I'd move this block to the top of the doc block.

And finish with something like: "Taken from xx and this is the original doc:"

Reply:
OK - done